src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] list = new Figure [6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i <= 2; i++) {
+            list[i] = figureSupplier.getRandomFigure();
+            System.out.println(list[i].toString());

```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. However, in this case, you are correctly creating `FigureSupplier` only once before the loop starts, which is good.